src/test/java/com/exasol/adapter/document/files/AbstractDocumentFilesAdapterIT.java

```diff
@@ -936,13 +936,8 @@ public void refreshVirtualSchemaReadsIgnoresNonSchemaChanges() {
                 .mapField("0", fieldMapping.overflowBehaviour(TruncateableMappingErrorBehaviour.TRUNCATE).build())
                 .build()), mappingFile);
         refreshVirtualSchema("TEST");
-        // This should succeed and return the truncate value but fails due to a bug in Exasol:
-        // https://exasol.my.site.com/s/article/Changelog-content-20991
-        final Exception exception = assertThrows(IllegalStateException.class,
-                () -> this.assertQuery("SELECT * FROM TEST.BOOKS",
-                        ResultSetStructureMatcher.table("VARCHAR").row("short").row("very_l").matches()));
-        assertThat(exception.getMessage(), containsString(
-                "OverflowException: E-VSD-38: A value for column 'EXA_COL' exceeded the configured varcharColumnSize of 6."));
+
+        this.assertQuery("SELECT * FROM TEST.BOOKS", table("VARCHAR").row("short").row("very_l").matches());
```

Comment:
I think the name `refreshVirtualSchemaReadsIgnoresNonSchemaChanges` of the test case still reflects the old semantics. Maybe we should rename it? 

Additionally, I wonder: How could / can the VSDF identify the version of the Exasol database? 
Or is this up to the concrete implementations inheriting from VSDF?

In any way: Can and should the VSDF at least signal, which versions of Exasol database are supported or which include the bug and which fix it?